CvGameCoreDLL_Expansion2/CvUnit.cpp

```diff
@@ -30283,770 +30295,771 @@ int CvUnit::AI_promotionValue(PromotionTypes ePromotion)
 			iValue += 1000;   // Enough to lock this one up
 		}
 	}
-
-#if defined(MOD_BALANCE_CORE_MILITARY_PROMOTION_ADVANCED)
+	*/
```

Comment:
Just ignore the red from here down. Everything is rearranged and all of the values added per promotion benefit have been changed.